src/test/java/io/conduit/TransformationsTest.java

```diff
@@ -1,285 +0,0 @@
-package io.conduit;
```

Comment:
Do we still need `src/main/java/io/conduit/Transformations.java`?

Reply:
It must have sneaked back in, after a merge. Good catch, I'll remove.